packages/core/theme/src/components/select.ts

```diff
@@ -9,9 +9,9 @@ const select = tv({
     label: "block text-small font-medium text-foreground-500 pointer-events-none",
     mainWrapper: "w-full flex flex-col",
     trigger:
-      "relative px-3 gap-3 w-full inline-flex flex-row items-center shadow-sm outline-none tap-highlight-transparent",
+      "relative px-3 gap-3 w-full inline-flex flex-row items-center shadow-sm outline-none tap-highlight-transparent pr-10",
```

Comment:
@jguddas let's reduce the pr to `6`, please check this line https://github.com/nextui-org/nextui/pull/1591/files#diff-cf57aaf38eadf5556e28ffe5f5cd170c1b121821a248be7d551d02c13025b0c4L88 

The `px` overrides the `pr-` on `sm` sizes